src/Converters.ts

```diff
@@ -34,9 +26,11 @@ export function toRpcHttp(inputMessage): rpc.TypedData$Properties {
 }
 
 export function toRpcHttpHeaders(inputHeaders: rpc.TypedData$Properties) {
-  let rpcHttpHeaders: {[key: string]: rpc.TypedData$Properties} = {};
+  let rpcHttpHeaders: {[key: string]: string} = {};
   for (let key in inputHeaders) {
-    rpcHttpHeaders[key] = toTypedData(inputHeaders[key]);
+    if (inputHeaders[key] != null) {
```

Comment:
if(inputHeaders[key]) should take care of undefined , null

Reply:
it evaluates to false with a value of 0 though :(

Reply:
Ahh. Thanks for checking. Add a test for 0 too.